types/react-flatpickr/index.d.ts

```diff
@@ -11,7 +11,7 @@ import flatpickr from "flatpickr";
 
 export type Omit<T, K extends keyof any> = Pick<T, Exclude<keyof T, K>>;
 
-export interface DateTimePickerProps extends Omit<Partial<HTMLInputElement>, 'children' | 'value'> {
+export interface DateTimePickerProps extends Omit<React.ComponentPropsWithoutRef<'input'>, 'children' | 'value' | 'onChange'> {
```

Comment:
This removes the `partial`; is that intended?

Reply:
Yes. It is not longer needed, because it's a react component and not an DOMInterface :)
Almost all properties/attributes are already optional.

See also the tests which passed